### PR TITLE
Fix saving config file for options with multiple arguments.

### PR DESCRIPTION
**Pull request by clydeclements**
Fix for issue #95.

### CODE REVIEW

**Comment by pltrdy** [comment]
@bw2 it would be cool to (i) merge this, (ii) make a new pip version if possible. 

Thanks.

**Comment by ferrine** [comment]
I have the same issue, ping @bw2

File: configargparse.py [hunk 1]
```diff
@@ -607,10 +607,6 @@ def get_items_for_config_file_output(self, source_to_settings,
                         if value is not None:
                             if isinstance(value, bool):
                                 value = str(value).lower()
-                            elif callable(action.type):
```
Comment (bw2) [inline configargparse.py]: This code was added to handle custom types (https://github.com/bw2/ConfigArgParse/issues/69)
Wouldn't taking it out break that again?

Comment (bw2) [reply configargparse.py]: Never mind, just read your comments on the new issue.

**Comment by bw2** [comment]
Thank you for the PR, and apologies for being MIA for this long. 
If you guys would be interested in joining the project as maintainers, let me know.

**Comment by ferrine** [comment]
Thanks! Is this already on pip?

**Comment by bw2** [comment]
yes, I uploaded a new version - v0.14.0

**Pull request merged**